packages/hunspell.rb

```diff
@@ -3,32 +3,26 @@
 class Hunspell < Package
   description 'Hunspell is a spell checker and morphological analyzer library'
   homepage 'http://hunspell.github.io/'
-  version '1.6.2'
-  source_url 'https://github.com/hunspell/hunspell/archive/v1.6.2.tar.gz'
-  source_sha256 '3cd9ceb062fe5814f668e4f22b2fa6e3ba0b339b921739541ce180cac4d6f4c4'
+  version '1.7.0'
+  source_url 'https://github.com/hunspell/hunspell/archive/v1.7.0.tar.gz'
+  source_sha256 'bb27b86eb910a8285407cf3ca33b62643a02798cf2eef468c0a74f6c3ee6bc8a'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell-1.6.2-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell-1.6.2-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell-1.6.2-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/hunspell-1.6.2-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '2122527ab9f428c6d88d22ea142c134bf648b933963178f368ce5e9832d93317',
-     armv7l: '2122527ab9f428c6d88d22ea142c134bf648b933963178f368ce5e9832d93317',
-       i686: '1e7bd3254580e4a211b1b8208e548675602290e9c7a1105ba66af6d4c9cee8da',
-     x86_64: '76e93eec7e0cec41707fd68f0f21351e94c25e8b993431f3bf400693d3de8251',
   })
 
   depends_on 'readline'
+  depends_on 'ncurses'
```

Comment:
No need to include either of these dependencies since they're already included in "core".  See https://github.com/skycocker/chromebrew/blob/master/tools/core_packages.txt.